websites/A/Apple Podcasts/presence.ts

```diff
@@ -0,0 +1,101 @@
+const presence = new Presence({
+		clientId: "1281538997235224596",
+		injectOnComplete: true,
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets { // Other default assets can be found at index.d.ts
+	Logo = "https://i.imgur.com/ibW2CvC.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		strings = await presence.getStrings({
+			play: "general.playing",
+			pause: "general.paused",
+		}),
+		[privacy] = await Promise.all([presence.getSetting("privacy")]),
+		shadowRootDir = document.querySelector(".lcd").shadowRoot;
+	// document.querySelectorAll(".player > .artwork-container > .artwork-component > picture > img")[0].currentSrc
+	if (shadowRootDir?.querySelector(".lcd__active").ariaHidden !== "true") {
+		presenceData.type = ActivityType.Listening;
+
+		const episodeImage = (
+				shadowRootDir.querySelectorAll(
+					".lcd__artwork > picture > img"
+				)[0] as HTMLImageElement
+			).src.replace("88x88", "450x450"),
+			playingShadow = document.querySelector(".chrome-player").shadowRoot;
+
+		presenceData.name = shadowRootDir.querySelectorAll(
+			".lcd-meta-line__fragment"
+		)[0].textContent;
```

Comment:
Don't use querySelectorAll if you are just going to get the first item, then just use querySelector